### DISCUSSION TITLE
Does Zephyr work alongside NXP HAB for secure booting?

### DISCUSSION THREAD

**Discussion by bobbychen19780505** [zephyrproject-rtos/zephyr#30580] [Q&A]
Hi guys, I tried to add the NXP HAB secure boot function to Zephyr bin file but failed.

My situation is: the code I built on MCUXpresso can perform a secure boot smoothly, but the code generated with Zephyr cannot. It looks like the XIP Header has not been removed. How to set up correctly in Zephyr?

@jhqian

**Comment by henrikbrixandersen** [comment]
Perhaps @MaureenHelm knows?
